tpm/tpm_test.go

```diff
@@ -232,6 +232,33 @@ func TestResizeableSlice(t *testing.T) {
 	}
 }
 
+func TestNVDefineSpace(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := tpmutil.Handle(0x40000001)
+	permissions := []uint32{nvPerWriteSTClear}
+	length := uint32(54)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	_, err := NVDefineSpace(rwc, index, length, permissions, ownerAuth)
+	if err != nil {
+		t.Fatal("Couldn't define space:", err)
+	}
+
+}
+
+func TestNVWriteValue(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := uint32(0x40000001)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	randomdata := make([]byte, 20)
```

Comment:
Go style insists on "randomData" over "randomdata" (since it's two words).

However, it would be even better in Go to call this simply `r`.